docs/editing-components.md

```diff
@@ -0,0 +1,376 @@
+        ('full_width_text', organisms.FullWidthText()),
+        ('info_unit_group', organisms.InfoUnitGroup()),
+        ('expandable', organisms.Expandable()),
+        ('well', organisms.Well()),
+        ('email_signup', organisms.EmailSignUp()),
+        ('feedback', v1_blocks.Feedback()),
+        ('image_text_50_50_group', organisms.ImageText5050Group()),
+    ])
+    …
+```
+
+This sets up a StreamField named `content` that allows for
+the insertion of any of those seven listed blocks into it.
+
+To make the `RelatedContent` module (shown above) available to this StreamField,
+we'd add a new entry to this list following the same format:
+`('related_content', molecules.RelatedContent()),`.
+
+Most page types have a couple StreamFields (`header` and `content`) in the
+general content area (the first tab on an editing screen), and most also share
+[a common `sidefoot` StreamField](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L95-L107)
+(so named for the fact that it appears on the right side on some page types,
+but in the footer on others) on the sidebar tab.
+
+
+---
+
+!!! note "Warning: Migrations may be required!"
+    Changes to either of the Python bits will usually require
+    a new schema migration file, and _may_ require a data migration.
+    See
+    ["Creating migrations for StreamField blocks"](#creating-migrations-for-streamfield-blocks)
+    below.
+
+With the Python pieces covered, it's time to talk about the front end.
+First: the markup.
+
+
+### The HTML template
+
+As mentioned above, each module's Python class has a `Meta` class
+that defines the location of its template file.
+In the `RelatedContent` example,
+it was `template = '_includes/molecules/related-content.html'`.
+That path is relative to `cfgov/jinja2/v1/`.
+
+This is what that template looks like (comments excluded):
+
+```html
+<div class="m-related-content">
+    {% if value.heading %}
+        <header class="m-slug-header">
+            <h2 class="a-heading">
+                {{ value.heading }}
+            </h2>
+        </header>
+    {% endif %}
+
+    {{ value.paragraph | safe }}
+
+    {% if value.links %}
+        <ul class="m-list m-list__links">
+        {% for link in value.links %}
+            <li class="m-list_item">
+                <a href="{{ link.url }}" class="m-list_link">{{ link.text }}</a>
+            </li>
+        {% endfor %}
+        </ul>
+    {% endif %}
+</div>
+```
+
+When Wagtail renders a block,
+it includes the values of its sub-blocks in an object named `value`.
+Above, you can see where the sub-blocks are output with
+[Jinja2 expression tags](http://jinja.pocoo.org/docs/2.10/templates/#expressions).
+In particular, note how the `links` `ListBlock` is iterated over,
+and the values of _its_ `Hyperlink` child blocks are output.
+
+That's about as simple an example as it gets, but block templates can get much
+more complex when they have lots of child blocks and grandchild blocks.
```

Comment:
This might be a place to mention custom `get_context` as mentioned above. Additionally, use of [`{% include_block %}`](http://docs.wagtail.io/en/v2.3/topics/streamfield.html#template-rendering) could be mentioned here to include child blocks.

Reply:
Addressed in a47039c